pages/windows/set-date.md

```diff
@@ -0,0 +1,17 @@
+# Set-Date
+
+> Changes the system time on the computer to a time that you specify.
+> This command can only be used through PowerShell.
+> More information: <https://learn.microsoft.com/powershell/module/microsoft.powershell.utility/set-date>.
+
+- Add three days to the system date:
+
+`Set-Date -Date (Get-Date).AddDays(3)`
+
+- Set the system clock back 10 minutes:
+
+`Set-Date -Adjust -0:10:0 -DisplayHint Time`
+
+- Add 90 minutes to the system clock:
+
+`$90mins = New-TimeSpan -Minutes 90; Set-Date -Adjust $90mins`
```

Comment:
```suggestion
`${{90}}mins = New-TimeSpan -Minutes {{90}}; Set-Date -Adjust ${{90}}mins`
```
or 

```suggestion
`$90mins = New-TimeSpan -Minutes {{90}}; Set-Date -Adjust $90mins`
```

I am not sure about this.

Reply:
The `$90mins` is the name of the variable. If it's a little ambiguous I can change to `{{$timespan}}`, what do you think?

Reply:
Does this parameter support seconds, hours,etc if so we can use timespan.

Reply:
It does, but on another parameter flags.

Eg: -Days, -Hours, -Seconds

Reply:
I guess we no need to change this command then.

Reply:
Ok, I'll just add the token `{{}}` to the values `{{3}}` and `{{90}}` then.